index.html

```diff
@@ -317,6 +317,25 @@ <h2 class="connect-link">LET'S CONNECT</h2>
           </li>
         </ul>
       </section>
+      <section class="fotter-contact-form-container">
+        <section class="fotter-contact-form-text">
+          <h2 class="fotter-contact-form-heading">Contact me</h2>
+          <p class="fotter-contact-form-subheading">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it!</p>
+             <form action="https://formspree.io/f/mdorkznl" method="post">
+          <input required="required" name="Name" id="Name"  maxlength="30"  placeholder="Lorem Ipsum" type="text">
+          <input required="required" name="email" id="email"   type="email" placeholder="loremipsum@gmail.com">
+          <textarea required="required" maxlength="500" placeholder="Write your message here.."></textarea>
+          <input class="btn-footer-container" value="Get in touch"   type="submit">
+          <section class="fotter-contact-form-imag"><img src="/assets/images/Shape.png" alt="fotter-background-image" class="footer-background-image"></section>
+        </form>
+        </section>
+        <section class="fotter-contact-form-imag"><img src="assets/images/contact form shapes-mobile.png" alt="fotter-background-image" class="footer-background-image"></section>
+
+      </section>
```

Comment:
- kindly as the previous reviewer suggested update the contact section `getintech` button and minimize the space and fix the broken image for the horizontal line 👍 
- 

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/52010789/2650efdd-0e63-49dd-87f1-fb3f1af9368e" width="200">|<img src="https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/52010789/1a9290bd-0a15-4f13-a7b8-3a5f46e54daa" width="200">|


😎️

🤝️

Reply:
can you check aggain please

Reply:
yup i see can you check so that I can send the review

Reply:
please can you see this is new one plz